Gemfile.jruby-1.9.lock

```diff
@@ -753,7 +755,7 @@ DEPENDENCIES
   logstash-output-http
   logstash-output-irc
   logstash-output-juggernaut
-  logstash-output-kafka
+  logstash-output-kafka (= 3.0.0.beta2)
```

Comment:
for alpha1, @suyograo thought it would be a good idea since the 3.x version of this will make it to 5.0